lib/chess_web/channels/game_channel.ex

```diff
@@ -94,4 +110,12 @@ defmodule ChessWeb.GameChannel do
       "black"
     end
   end
+
+  defp opponent(socket, game) do
```

Comment:
Is this logic duplicated elsewhere? Seems like I've seen this before. I know it's _similar_ at least, is there a way to abstract it?